### PR TITLE
Add infrastructure to apply outstanding PT PRs.

### PR DESCRIPTION
**Pull request by dlibenzi**

### CODE REVIEW

File: .circleci/build.sh [hunk 1]
```diff
@@ -34,7 +34,7 @@ pip install ninja
 pip install lark-parser
 
 # Install Pytorch
-patch -p1 < xla/pytorch.patch
+xla/scripts/apply_patches.sh
```
Comment (asuhan) [inline .circleci/build.sh]: Doesn't `kokoro/ubuntu/common.sh` need the same change?

Comment (dlibenzi) [reply .circleci/build.sh]: Yeah

**Comment by dlibenzi** [comment]
@ailzhang yes, looks like it :)

**Comment by ailzhang** [comment]
@dlibenzi Given 16585 is in master now, do we still want to proceed and land this PR(after removing the unncessary patch)?

**Comment by dlibenzi** [comment]
PR patch removed.

**Comment by dlibenzi** [comment]
@asuhan I think so. It cleans up the patch apply process, and covers us from future delays.

**Comment by dlibenzi** [comment]
@ailzhang Do I have to merge after your PR, or before?

**Comment by ailzhang** [comment]
@dlibenzi  Please feel free to merge it once the test here is green. I will merge my PR later to get Pytorch CI passes.

**Pull request merged**